src/SprykerEco/Zed/Computop/Business/Payment/Handler/Saver/Complete/PayPalExpressCompleteResponseSaver.php

```diff
@@ -109,8 +109,10 @@ protected function savePaymentComputop(
         ComputopApiPayPalExpressCompleteResponseTransfer $computopApiPayPalExpressCompleteResponseTransfer,
         ComputopPaymentComputopTransfer $computopPaymentComputopTransfer
     ): void {
-        $computopPaymentComputopTransfer->setPayId($computopApiPayPalExpressCompleteResponseTransfer->getHeader()->getPayId());
-        $computopPaymentComputopTransfer->setXId($computopApiPayPalExpressCompleteResponseTransfer->getHeader()->getXId());
+        $header = $computopApiPayPalExpressCompleteResponseTransfer->getHeaderOrFail();
+
+        $computopPaymentComputopTransfer->setPayId($header->getPayIdOrFail());
```

Comment:
Can we use method chaining here?
Please check also similar places.
e.g.
```
$computopPaymentComputopTransfer->setPayId($header->getPayIdOrFail())
    ->setXId($header->getXIdOrFail());

Reply:
Fixed